submit_reports/models.py

```diff
@@ -106,7 +116,12 @@ def __unicode__(self):
 # Data Classes
 ######################################################
 
+class SubmitReportManager(models.Manager):
+	def query_pending_reports(request):
+		return SubmitReport.objects.all().filter(status="PENDING")
```

Comment:
according to this link we should be adding queries to managers. @msuk how do you feel about this?